### PR TITLE
Show nav_divider image using CSS rather than HTML

### PR DESCRIPTION
**Pull request by PatrickMcSweeny**
Closes exercism/exercism#4626. Screenshot:

screenshot from 2019-01-22 16-49-35

### CODE REVIEW

File: app/assets/stylesheets/nav-bar.scss [hunk 1]
```diff
@@ -23,23 +23,20 @@
         text-decoration:none;
         font-weight:$regular;
         position:relative;
-        &:before {
```
Comment (iHiD) [inline app/assets/stylesheets/nav-bar.scss]: Why has this block been removed?

Comment (PatrickMcSweeny) [reply app/assets/stylesheets/nav-bar.scss]: It didn't serve any purpose as far as I could tell and I didn't want it to cause any conflict with the new approach.

Comment (iHiD) [reply app/assets/stylesheets/nav-bar.scss]: It underlines the link. 

Before: 
screenshot 2019-01-23 at 11 42 10

After:
screenshot 2019-01-23 at 11 42 13

Comment (PatrickMcSweeny) [reply app/assets/stylesheets/nav-bar.scss]: Good catch. I restored that code.

**Comment by iHiD** [comment]
Closing in favour of https://github.com/exercism/website/pull/528 because I can't push to this branch for some reason.

**Pull request closed without merging**